shell/platform/android/test/io/flutter/embedding/android/FlutterFragmentActivityTest.java

```diff
@@ -127,6 +129,24 @@ public void itReturnsValueFromMetaDataWhenCallsShouldHandleDeepLinkingCase3()
     assertFalse(spyFlutterActivity.shouldHandleDeeplinking());
   }
 
+  @Test
+  public void itAllowsRootLayoutOverride() {
+    FlutterFragmentActivityWithRootLayout activity =
+        Robolectric.buildActivity(FlutterFragmentActivityWithRootLayout.class).get();
```

Comment:
See http://robolectric.org/androidx_test and setupActivity vs buildActivity. If you're not doing something special with the lifecycle stages it goes through, just run setupActivity.

Reply:
That won't work.
The created activity cannot be started since the FlutterFragment created in FlutterFragmentActivityWithProvidedEngine is not correctly hooked up.

Reply:
alrighty :) we'll look after. Thanks for the PR!